.rubocop_todo.yml

```
@@ -25,7 +25,11 @@ InternalAffairs/NodeDestructuring:
 # Offense count: 10
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
-  Max: 163
+  Max: 139
+  Exclude:
+    - 'lib/rubocop/cop/rails/time_zone.rb'
+    - 'lib/rubocop/cop/rails/save_bang.rb'
+    - 'lib/rubocop/cop/rails/file_path.rb'
```

It is concerning that unrelated files are specified in `Exclude`. Can you update the value of `Max` for now?

```suggestion
  Max: 179
```

---

done